src/api/test_helpers.ts

```diff
@@ -32,6 +32,9 @@ export async function setupEnvironment(): Promise<Partial<IRunEnvironment>> {
 }
 
 export async function teardownEnvironment(environment: IRunEnvironment) {
-  await fs.rmdir(environment.cwd, { recursive: true })
-  environment.stdout.end()
+  return new Promise((resolve) => {
```

Comment:
Apparently mx/fs doesn't have the new version of fs.rm in its implementation - that or it's incompatible with the old one the deprecation warns about. In any event, to resolve the deprecation I had to resort to the older Promise.then pattern.